tests/hopr.py

```diff
@@ -352,3 +362,16 @@ async def ticket_price(self):
         """
         _, response = self.__call_api(NetworkApi, "price")
         return int(response.price) if hasattr(response, "price") else None
+
+    async def startedz(self):
+        """
+        Checks if the node is started.
+        :return: started: int
+        """
+        self.__call_api(ChecksApi, "startedz")
```

Comment:
It'd be better to check for status code and return a boolean here.

Reply:
Status code is not accessible with the generated API. The methods now returns the status returned by `__call_api` which is true if no errors are thrown by the API.

Reply:
The API returns `tuple[bool, Optional[object]]`. That `Optional[object]` is a http response, it should contain the response code. If not, it could be added.